kandji2snipe

```diff
@@ -121,16 +122,16 @@ def get_settings():
         logging.error('Invalid Kandji Region, check your settings.conf and try again.')
         sys.exit(exit_error_message)
 
-    if config['kandji']['apitoken'] == "kandji-api-bearer-token-here" or config['kandji']['apitoken'] == "":
-        logging.error('Invalid Kandji API Token, check your settings.conf and try again.')
+    if config['kandji']['apitoken'] == "kandji-api-bearer-token-here" or config['kandji']['apitoken'] == "" or os.environ.get("KANDJI_APITOKEN") == "":
```

Comment:
Can you take another look at this logic? Using an `or` here, this will error if _any_ of these things are true.... which means you'll always have to provide an ENV var _and_ a config value....